evaluator/src/test/kotlin/EvaluatorTest.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.evaluator
+
+import com.here.ort.model.OrtResult
+import com.here.ort.model.Repository
+import com.here.ort.model.VcsInfo
+import com.here.ort.model.config.RepositoryConfiguration
+import io.kotlintest.matchers.containExactly
+import io.kotlintest.matchers.containsInOrder
+import io.kotlintest.matchers.haveSize
+import io.kotlintest.should
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.WordSpec
+
+class EvaluatorTest : WordSpec() {
+    init {
+        val ortResult = OrtResult(Repository(VcsInfo.EMPTY, VcsInfo.EMPTY, RepositoryConfiguration()))
+
+        "checkSyntax" should {
+            "succeed if the script can be compiled" {
+                val result = Evaluator().checkSyntax(ortResult, "")
```

Comment:
Maybe use the contained `no_gpl_declared.kts` here to also ensure this embedded rule compiles?

Reply:
Good idea, done.